game.cpp

```diff
@@ -1760,39 +1759,29 @@ void test_game() //!OCLINT tests may be many
     int n_enemies = 0;
     int n_food = 1;
 
-    game g(short_wall_side); // just for getting min and max coordinates below
-
-    int repeats = 1000;
-    for(int rng_seed = 0; rng_seed != repeats; rng_seed++)
-      {
-        game g(short_wall_side,
-               n_players,
-               n_ticks,
-               n_shelters,
-               n_enemies,
-               n_food,
-               rng_seed
-               );
-        food_x.push_back(get_nth_food_x(g, 0));
-        food_y.push_back(get_nth_food_y(g, 0));
-      }
-    auto mean_x = calc_mean(food_x);
-    auto mean_y = calc_mean(food_y);
-
-    // Mean position of food items should be center of game environment
-    double expected_mean_x = (get_max_x(g) + get_min_x(g)) / 2.0;
-    double expected_mean_y = (get_max_y(g) + get_min_y(g)) / 2.0;
-    double d_mean_x = abs(expected_mean_x - mean_x);
-    double d_mean_y = abs(expected_mean_y - mean_y);
-
-    assert(d_mean_x < 0.01);
-    assert(d_mean_x < 0.01);
+    int a_seed = 1;
```

Comment:
A tiny detail: it could be `const int a_seed = 1;`, hence it should be a `const`, see [C++ Core Guideline 'Con.1: By default, make objects immutable'](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#con1-by-default-make-objects-immutable)